src/data/redux/hooks/requests.js

```diff
@@ -29,15 +29,20 @@ export const useMakeNetworkRequest = () => {
       if (onSuccess) { onSuccess(response); }
       dispatch(actions.completeRequest({ requestKey, response }));
     }).catch((error) => {
-      if (onFailure) { onFailure(error); }
-      dispatch(actions.failRequest({ requestKey, error }));
+      // Parse backend error from axios object
```

Comment:
As is this is always going to be an axios object but in the event we do some custom error parsing before we get here the error will pass through.